src/Diagnostics.ModelsAndUtils/ScriptUtilities/Anonymizer.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
```

Comment:
Puneet do you think if customers accidentally put secrets here it will be handled with these 3 methods?

Reply:
Secrets explicitly won't be handled. This is just a wait to avoid showing PII (Email, phone numbers, digits) etc in exception messages in detectors shown in Applens. (Someone reached out mentioning that some customer's email addresses were being revealed here). 

This is just one small way of masking that information to Applens users.

Reply:
We can come up with ways to hide secrets too in future in this class but the existing logic is not complete yet.